React Project/src/pages/Login/Login.tsx

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import { Link } from "react-router-dom";
+import TextField from "../../components/TextField";
+import Button from "../../components/Button";
+import { encrypt } from "@omar-sarfraz/caesar-cipher";
+import { User } from "../SignUp/SignUp";
+import bcrypt from "bcryptjs";
+
+export type LoginError = {
+    type: "" | "EMAIL" | "PASSWORD";
+    message: string;
+};
+
+export default function Login() {
+    const [email, setEmail] = useState<string>("");
+    const [password, setPassword] = useState<string>("");
+    const [error, setError] = useState<LoginError>({ type: "", message: "" });
+
+    const handleLogin = async () => {
+        setError({ type: "", message: "" });
+
+        if (!email) {
+            setError({ type: "EMAIL", message: "Email is required!" });
+            return;
+        }
+
+        if (!password) {
+            setError({ type: "PASSWORD", message: "Password is required!" });
+            return;
+        }
+
+        const key = parseInt(import.meta.env.VITE_CIPHER_KEY);
+        const encryptedEmail = encrypt(key, email);
```

Comment:
why aren't you using the cezar-cypher package you created here?

Reply:
I've used the caesar-cipher package to encrypt name and email. For passwords, i am creating an hash as it is more secure. The encrypt function is from the caesar-cipher package.